### ISSUE TITLE
Moving Beyond PubMed

### ISSUE THREAD

**Issue by karafecho** [NCATS-Gamma/robokop#351]
As we consider new collaborations in the social, political, and educational sciences, I think it might be helpful to consider new indexing systems for publications. Even within the biomedical domain, PubMed often is insufficient. For instance, I have articles related to patient safety that aren't indexed by PubMed.

**Comment by cbizon** [comment]
Agreed - seems to me that the most difficult component here is trying to the indexing services.   In some fields they exist but are for license (like inspec) and maybe dont' exist for other fields.

Is google scholar a viable option?  I don't know any way to get abstracts or a data dump from it.
